### DISCUSSION TITLE
Syntax to specify time grain for field for JSON API

### DISCUSSION THREAD

**Discussion by justin-tay** [yahoo/elide#2760] [Q&A] [answered]
What is the syntax to specify an attribute for a field for JSON API? 

`/playerStats?fields[playerStats]=gameOn,highScore&sort=highScore`

For instance if I want to get the `gameOn` field to use the MONTH time grain.

For the filter I see that using `gameOn[grain:MONTH]=='2020-08'` in the filter seems to work but this same syntax in the sparse fieldset does not work.

**Comment by aklish** [comment, marked as answer]
JSON-API unfortunately doesn't support parameterized attributes.  That is only a feature in GraphQL.   You would need to create separate attributes in your model for each time grain in JSON-API (single grain per attribute).
